components/dot_menu/dot_menu.tsx

```diff
@@ -26,73 +25,79 @@ import DotsHorizontalIcon from 'components/widgets/icons/dots_horizontal';
 const MENU_BOTTOM_MARGIN = 80;
 
 export const PLUGGABLE_COMPONENT = 'PostDropdownMenuItem';
+type Props = {
+    post: any;
+    teamId?: string;
+    location?: 'CENTER' | 'RHS_ROOT' | 'RHS_COMMENT' | 'SEARCH' | string; // TechDebt: Made non-mandatory while converting to typescript
```

Comment:
im not sure of the syntax to get this done - could you please make the changes?

Reply:
It seems typescript doesn't like using imported constants as type definitions. I think this if fine for now. I suppose this will be fixed when utils/contants.jsx is migrated to typescript.

Reply:
yes - left the `TODO` in place for future reference.